fuzzers/neuzz/fuzzer.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for AFL fuzzer."""
+
+import os
+import shutil
+import subprocess
+import time
+import signal
+
+from fuzzers import utils
+
+WARMUP = 60 * 60
+
+
+def prepare_build_environment():
+    """Set environment variables used to build targets for AFL-based
+    fuzzers."""
+    cflags = ['-O2', '-fno-omit-frame-pointer']
+    # ASAN?
```

Comment:
This is problematic, can we not use existing flags. Setting your custom flags would mean it wont work with bug based benchmarks (they use ASan). It is better to remove/change part of flags that dont work (e.g. optimization flag).

Reply:
done,